src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -255,7 +256,16 @@ public boolean hasTag(String tag) {
 	}
 
 	private static @NotNull DefaultExpression<?> getDefaultExpression(ExprInfo exprInfo, String pattern) {
-		DefaultExpression<?> expr = exprInfo.classes[0].getDefaultExpression();
+		DefaultExpression<?> expr = null;
+		// check custom default values first.
+		DefaultValueData data = getParser().getData(DefaultValueData.class);
+		if (data != null)
+			expr = data.getDefaultValue(exprInfo.classes[0].getC());
```

Comment:
```suggestion
		DefaultValueData data = getParser().getData(DefaultValueData.class);
		if (data != null)
			expr = data.getDefaultValue(exprInfo.classes[0].getC());
```
I don't believe `data` will ever be null. `getData()` is not null/will create a new instance if it does not exist